doc/ideas/fr_technical_overview.md

```diff
@@ -0,0 +1,137 @@
+GAIA               | G5         | Renewable resources such as trees and animals that repopulate. Forest regeneration or seeding.
+GAIA               | G6         | Buildings in later ages cost stone with possibility of deep mining stone. villagers can dig/mine natural resources.
+GAIA               | G7         | Better fauna (Tapir, Tiger, ...). Improved animal colours
+GAIA AI            | GI1        | Fish migrate and replenish.
+GAIA AI            | GI2        | Mercenaries could be hired on behalf of the player
+Gameplay           | GM1        | Roads on commonly traveled areas allow faster traffic. Pathfinding would let the units stay on the road. Appearance and quality depends on traffic (matted grass, dirt path, etc.). Neutral to all players. turns Soft/Regular Terrains into Roads, requires Stone, does not work on Irregular Terrains; grants Movement speed to all Allied Units + Passive vision (like a foundation spots units walking over them).
+Gameplay           | GM2        | Bridges for connections between islands. blocks ships, allows land units to walk on. terraforming. wooden and stone bridges with different hp.
+Gameplay           | GM3        | Canals: creates empty, shallow and navigatable water.
+Gameplay           | GM4        | single units can hide/bunk in the forest for sneak attacks, only visable to a certain type of unit. garrison function with no symbol, you can loose your units if you don't remember them.
+Gameplay           | GM5        | New Unit: spies (can burn down your city), lame your production
+Gameplay           | GM6        | New Unit: monk-ship on water to convert ships
+Gameplay           | GM7        | New Unit: transport waggon on land to slowly transport ships over land
+Gameplay           | GM8        | villagers need sleep at night, die at work (New Building: Caffeine industry)
+Gameplay           | GM9        | GAIA and terrain interaction, e.g. flock of bird rises up from tree when tree is felled or many people approach, hearable sound indicator, players hear and see birds when someone comes to their woodline at home. triggered rocks falling of cliffs.
+Gameplay           | GM10       | Relics have special abilities (AOM), bonuses for attack, range. generated randomly. maybe also rolling relics with ranged attributes to bring into battle.
+Gameplay           | GM11       | Bring Kings/Queens into battle for special abilities within some tiles radius.
+Gameplay           | GM12       | Building produces in loop (AOM) as long as resources and pop space are available. units and farms. indicator over buildings or in the GUI for loop production. Toggle reseeding on/off.
+Gameplay           | GM13       | Mills can produce sheep, cows, turkeys
+Gameplay           | GM14       | Researching faith in third age allows to convert units, before monk can heal and carry relics.
```

Comment:
"Conversion is enabled by researching Faith. Before, monks can only carry relics and heal units."

Reply:
done